File: desktop/src/ru/icarumbas/bagel/desktop/DesktopLauncher.java
package ru.icarumbas.bagel.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;

import ru.icarumbas.Bagel;


public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		new LwjglApplication(new Bagel(), config);
		config.width = 800;
		config.height = 600;
	}
}

File: desktop/src/ru/icarumbas/bagel/desktop/DesktopLauncher.kt
package ru.icarumbas.bagel.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import org.jetbrains.annotations.Nullable
import ru.icarumbas.Bagel


object DesktopLauncher {
    @Nullable @JvmStatic @Override
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration()
        LwjglApplication(Bagel(), config)
        config.width = 800
        config.height = 480
    }
}

